### ISSUE TITLE
pick does not work if input is xml

### ISSUE THREAD

**Issue by balki** [mikefarah/yq#1284]
Version of yq: 4.26.1
Operating system: linux
Installed via: snap

**Input XML**

ex.xml:
```xml
<feed>
  <entry>
    <id>b9a1ef89</id>
    <title> Added NDJSON decoder (#1281) </title>
  </entry>
  <entry>
    <id>c2c48bbe</id>
    <title> Bumping version </title>
  </entry>
</feed>
```

**Command**

```
❯ cat ex.xml | yq -p xml '.feed.entry[] |= pick(["title"])' 
Error: cannot pick indicies from type  (feed.entry.0)
```
But this works:
```
❯ cat ex.xml | yq -p xml . | yq '.feed.entry[] |= pick(["title"])'
feed:
  entry:
    - title: Added NDJSON decoder (#1281)
    - title: Bumping version
```

**Comment by mikefarah** [comment]
Thanks for raising, I can replicate this issue - will have a fix next release

**Comment by balki** [comment]
Great! Thanks

**Comment by mikefarah** [comment]
Fixed in 4.27.1 :)
